discord/opus.py

```diff
@@ -57,6 +64,15 @@ class SignalCtl(TypedDict):
     music: int
 
 
+class EncoderKwargs(TypedDict, total=False):
+    application: APPLICATION_CTL
```

Comment:
I don't really see much point in allowing this in particular to be set?

Reply:
Without `total=False` pyright complains if you don't have all the parameters passed in the function, if that's what you were talking about.

Reply:
No I was talking about the application parameter.

Reply:
That was originally the only constructor parameter that Encoder had so it didn't occur to me to either remove it or arbitrarily restrict access to only that setting.